nixos/modules/virtualisation/anbox.nix

```diff
@@ -0,0 +1,50 @@
+# Systemd services for anbox.
+
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.virtualisation.anbox;
+
+in
+
+{
+  ###### interface
+
+  options.virtualisation.anbox = {
+    enable = mkEnableOption "Anbox Container Management Daemon";
+  };
+
+  ###### implementation
+
+  config = mkIf cfg.enable {
+    virtualisation.lxc.enable = true;
+
+    environment.systemPackages = [ pkgs.anbox.exe ];
+
+    services.udev.extraRules = ''
+      KERNEL=="binder", MODE="0666"
+      KERNEL=="ashmem", MODE="0666"
+    '';
+
+    systemd.services.anbox-container-manager = {
+      description = "Anbox Container Management Daemon";
+
+      wantedBy = [ "multi-user.target" ];
+      after = [ "systemd-udev-settle.service" ];
+
+      preStart = ''
+        ${pkgs.kmod}/bin/insmod ${pkgs.anbox.ashmem}/ashmem_linux.ko
+        ${pkgs.kmod}/bin/insmod ${pkgs.anbox.binder}/binder_linux.ko
```

Comment:
This could will problematic if booted kernel != current kernel. For that reason we usually boot.extraModulePackages.